.pre-commit-config.yaml

```diff
@@ -31,13 +31,23 @@ repos:
     rev: 22.8.0
     hooks:
     -   id: black
--   repo: https://github.com/asottile/reorder_python_imports
-    rev: v3.8.2
+-   repo: https://github.com/PyCQA/isort
+    rev: 5.10.1
     hooks:
-    -   id: reorder-python-imports
+    -   id: isort
+        args: ["--profile", "black", "--filter-files"]
 -   repo: https://github.com/asottile/pyupgrade
     rev: v2.37.3
     hooks:
     -   id: pyupgrade
+-   repo: https://github.com/PyCQA/flake8
+    rev: 4.0.1
+    hooks:
+    -   id: flake8
+        args: ["--max-line-length", "88", "--extend-ignore", "E203,E722"]
```

Comment:
Good point, I have similar settings in my editor. Can we also ignore W503, which violates PEP8 and is also possibly scheduled for deprecation: https://www.flake8rules.com/rules/W503.html